src/studio-home/tabs-section/utils.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Alphabetical sorting for arrays of courses and libraries.
+ *
+ * @param {array} arr - Array of courses or libraries.
+ * @returns {array} - An array of alphabetically sorted courses or libraries.
+ */
+const sortAlphabeticallyArray = (arr) => [...arr]
```

Comment:
What I am sure about is such utils are the must for unit testing.

Reply:
Thanks
Added new tests